documentation/shared/attributes.adoc

```diff
@@ -112,6 +112,7 @@
 //distributed tracing versions and links
 :OpenTelemetryAlphaVersion: 1.19.0-alpha
 :OpenTelemetryVersion: 1.19.0
+:OpenTelemetryKafkaClient: 2.6
```

Comment:
we use this in the otel dependencies:

![image](https://github.com/strimzi/strimzi-kafka-operator/assets/47596553/88aa9614-16cc-416d-9542-79721aa0b08a)

Reply:
No we should stick with the 1.19 one. That is the OpenTelemetry version. The 2.6 is the Kafka version instead. I think they had one for the older 0.x Kafka and then moved to 2.x (which is compatible with all the new ones 3.x).
It's something I have on my to-do list if we should investigate to update the Otel Kafka instrumentation library with a newer Kafka version but it would mean more workload on us. I don't know about the impact and the time needed for it.